src/test/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/XGBoostEvidenceFilterUnitTest.java

```diff
@@ -0,0 +1,427 @@
+                predictYProbaSpark, predictYProbaSerial, 0.0);
+    }
+
+    @Test(groups = "sv")
+    protected void testResourceXGBoostClassifier() {
+        // load classifier from resource
+        final Predictor resourcePredictor = XGBoostEvidenceFilter.loadPredictor(resourceClassifierModelFile);
+        final double[] resourceYProba = predictProba(resourcePredictor, classifierAccuracyData.features);
+        // check that predictions from resource are identical to local
+        AssertJUnit.assertArrayEquals("Predictions via loading predictor from resource is not identical to local file",
+                resourceYProba, predictYProbaSerial, 0.0);
+    }
+
+    @Test(groups = "sv")
+    protected void testFeatureConstruction() {
+        final XGBoostEvidenceFilter evidenceFilter = new XGBoostEvidenceFilter(
+                evidenceList.iterator(), readMetadata, params, emptyCrossingChecker
+        );
+        for(int ind = 0; ind < featuresTestData.stringReps.length; ++ind) {
+            final BreakpointEvidence evidence = evidenceList.get(ind);
+            final String stringRep = featuresTestData.stringReps[ind];
+            final EvidenceFeatures fVec = featuresTestData.features[ind];
+
+            final BreakpointEvidence convertedEvidence = BreakpointEvidence.fromStringRep(stringRep, readMetadata);
+            final String convertedRep = convertedEvidence.stringRep(readMetadata, params.minEvidenceMapQ);
+            AssertJUnit.assertEquals("BreakpointEvidence.fromStringRep does not invert BreakpointEvidence.stringRep",
+                    stringRep.trim(), convertedRep.trim());
+            final EvidenceFeatures calcFVec = evidenceFilter.getFeatures(evidence);
+            AssertJUnit.assertArrayEquals("Features calculated by XGBoostEvidenceFilter don't match expected features",
+                    fVec.getValues(), calcFVec.getValues(), featuresTol);
+        }
+    }
+
+    @Test(groups = "sv")
+    protected void testFilter() {
+        final List<BreakpointEvidence> expectedPassed = new ArrayList<>();
+        int index = 0;
+        for(final BreakpointEvidence evidence : evidenceList) {
+            final double proba = featuresTestData.proba[index];
+            if(proba > params.svEvidenceFilterThresholdProbability) {
+                expectedPassed.add(evidence);
+            }
+            index += 1;
+        }
+
+        final XGBoostEvidenceFilter evidenceFilter = new XGBoostEvidenceFilter(
+                evidenceList.iterator(), readMetadata, params, emptyCrossingChecker
+        );
+        final List<BreakpointEvidence> passedEvidence = new ArrayList<>();
+        evidenceFilter.forEachRemaining(passedEvidence::add);
+
+        AssertJUnit.assertEquals("Evidence passed by XGBoostEvidenceFilter not the same as expected",
+                expectedPassed, passedEvidence);
+    }
+
+    private static double[] predictProba(final Predictor predictor, final FVec[] testFeatures) {
```

Comment:
I like "prob" better than "proba"

Reply:
Renamed all "Proba" variables to "Proability"